core/domain/feedback_services_test.py

```diff
@@ -84,6 +85,7 @@ class SuggestionQueriesUnitTests(test_utils.GenericTestBase):
     THREADS = []
     THREADS_WITH_SUGGESTIONS = []
     SUGGESTIONS = []
+    TIME_IN_MILLIS = 12345678
```

Comment:
Actually, CURRENT_TIME_IN_MSEC would be better. I was wondering what this was the time 'of' (e.g. is it the time taken to do some action?)